src/utils/CompilerLocationUtils.fs

```diff
@@ -12,7 +12,7 @@ open System.Runtime.InteropServices
 module internal FSharpEnvironment =
 
     /// The F# version reported in the banner
-    let FSharpBannerVersion = "10.4.0 for F# 4.6"
```

Comment:
Was there a 10.5?

Reply:
Not yet, but VS2019.2 will have one.   https://github.com/microsoft/visualfsharp/blob/release/dev16.2/eng/Versions.props#L21